doc/generate-api-docs.sh

```diff
@@ -49,12 +49,30 @@ LIBS_SORTED=$(
     | egrep '^'$(echo $LIBS|sed 's/ /|/g')'$'
 )
 
+dependencies=''
+
 for LIB in $LIBS_SORTED; do
     cd $LIB
+    version=$(cat Cargo.toml | grep version | head -n1 | cut -d'"' -f2)
```

Comment:
As discussed, getting version from latest git tag should be the way, this will resolve the issue with this fix around enclave-runner.